From 3302eda283020b2c6f03127906a5f61de845e8cb Mon Sep 17 00:00:00 2001
From: Han Xu <han.xu@nxp.com>
Date: Fri, 10 Jun 2016 11:04:59 -0500
Subject: [PATCH 1087/5242] MLK-12897: ARM: dts: add one more dts for all qspi
 chip support on i.MX6ULL

commit  ab19fc57259140aa3744716959f7641ea4fde3d8 from
https://source.codeaurora.org/external/imx/linux-imx.git

change the qspi dts for default one qspi scenario and add one more for
all four qspi chip support.

Signed-off-by: Han Xu <han.xu@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/Makefile                         |    1 +
 .../boot/dts/imx6ull-14x14-ddr3-arm2-qspi-all.dts  |   18 ++++++++++++++++++
 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-qspi.dts |    9 ---------
 3 files changed, 19 insertions(+), 9 deletions(-)
 create mode 100644 arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-qspi-all.dts

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index 96410e1..6c89e67 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -610,6 +610,7 @@ dtb-$(CONFIG_SOC_IMX6ULL) += \
 	imx6ull-14x14-ddr3-arm2-lcdif.dtb \
 	imx6ull-14x14-ddr3-arm2-ldo.dtb \
 	imx6ull-14x14-ddr3-arm2-qspi.dtb \
+	imx6ull-14x14-ddr3-arm2-qspi-all.dtb \
 	imx6ull-14x14-ddr3-arm2-tsc.dtb \
 	imx6ull-14x14-ddr3-arm2-gpmi-weim.dtb \
 	imx6ull-14x14-ddr3-arm2-uart2.dtb \
diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-qspi-all.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-qspi-all.dts
new file mode 100644
index 0000000..a4cdbe5
--- /dev/null
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-qspi-all.dts
@@ -0,0 +1,18 @@
+/*
+ * Copyright (C) 2016 Freescale Semiconductor, Inc.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation.
+ */
+
+#define REWORKED_ENABLE_ALL_QSPI
+#include "imx6ull-14x14-ddr3-arm2.dts"
+
+&gpmi {
+	status ="disabled";
+};
+
+&usdhc2{
+	status ="disabled";
+};
diff --git a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-qspi.dts b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-qspi.dts
index 63659bd..2155c7a 100644
--- a/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-qspi.dts
+++ b/arch/arm/boot/dts/imx6ull-14x14-ddr3-arm2-qspi.dts
@@ -6,13 +6,4 @@
  * published by the Free Software Foundation.
  */
 
-#define REWORKED_ENABLE_ALL_QSPI
 #include "imx6ull-14x14-ddr3-arm2.dts"
-
-&gpmi {
-	status = "disabled";
-};
-
-&usdhc2 {
-	status = "disabled";
-};
-- 
1.7.9.5

